CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+#### Update the HTML for the character count
+
+We've updated the HTML for the character count.
+
+The component wrapper `data-module="govuk-character-count"` and its form group `class="govuk-form-group"` are now combined as the same `<div>`. The hint text used as the count message is now placed directly after the `<textarea>`.
+
+If you're not using Nunjucks macros, then you should:
+
+- move the `govuk-form-group` class from the form group to the component wrapper `<div>`
+- move the `govuk-form-group--error` class (optional) from the form group to the component wrapper `<div>`
+- move any other custom classes and attributes from the form group to the component wrapper `<div>`
+- remove the opening `<div>` and closing `</div>` tags previously used by the form group
+
+For example, see how the HTML below:
+
+```html
+<div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="100">
+  <div class="form-group form-group--error">
```

Comment:
Should we make these classes match the steps above? (ie: `govuk-form-group` and `govuk-form-group--error`)

Reply:
🤦‍♂️